Room, Level and Tower init

Heratexx opened the pull request:
Optional:
fixes #34

Room, Level and Tower with print methods

Heratexx committed:
Merge pull request #4 from SWE-C3/master

update Fork

Heratexx committed:
Room, Level and Tower init

Room, Level and Tower with print methods

Heratexx committed:
Merge branch 'master' into master

KnutZuidema commented on src/room.py:
```diff
@@ -0,0 +1,55 @@
+    """
+    def __init__(self,
+                 doors,
+                 event: None):
```
Falls du hier einen default Wert festlegen willst, sollte das so aussehen: `def __init__(self, doors, event=None)`

KnutZuidema commented on src/room.py:
```diff
@@ -0,0 +1,55 @@
+            mid += " "
+        else:
+            mid += self.event
+        if self.doors['o'] is False:
```
Für die eastern door vielleicht ein "e" statt einem "o"?

KnutZuidema commented on src/room.py:
```diff
@@ -0,0 +1,55 @@
+        :param x_pos: x position in the window
+        :return:
+        """
+        if self.doors['n'] is False:
```
`bool is False` kann man einfach als `not bool` schreiben

KnutZuidema commented on src/tower.py:
```diff
@@ -0,0 +1,21 @@
+        :return:
+        """
+        for level in self.levels:
+            level.print(window)
```
Werden hier nicht einfach alle Level übereinander geschrieben? Wann müssen wir denn mehr als ein Level auf einmal ausgeben?

Heratexx replied:
Ich wollte eigentlich im tower garkeine print haben. PyLint hat aber gemeckert weil es nicht genügen public methonden gab

Heratexx replied:
habe aber schon ne idee für einen Fix.

KnutZuidema replied:
Naja, wenn du keine Methoden für den Tower hast, dann muss es keine Klasse sein

Heratexx replied:
Ja das Stimmt natürlich. 
Das Spiel haben wir aber auch nicht voll durchgeplant da könnte sich auch noch Nutzen ergeben,, ich denke die Klasse ist schon in Ordnung. habe das jetzt auch umgebaut, man kann jetzt eine gewünschte Ebene über den Index printen.

Heratexx committed:
room and tower fixes

changed check for the eastern door
changed tower print to print just a given level

Heratexx committed:
Merge branch 'master' of github.com:Heratexx/Adventure-Game

KnutZuidema committed:
Merge branch 'master' into master
